sdk/storage/Azure.Storage.Files.DataLake/README.md

```diff
@@ -179,6 +179,37 @@ fileClient.SetAccessControlList(accessControlList);
 PathAccessControl accessControlResponse = fileClient.GetAccessControl();
 ```
 
+### Set and modify Access Controls (ACLs) on a DataLake Directory structure
```

Comment:
Should this be in the README?  Feels like something less important than Upload/Donwload/etc.

Reply:
The non recursive get/set ACL is in this readme, so that's why I added it.
Also, after spending some time with this feature I find it not trivial to understand how this works. Especially Remove ACL part where ACL string format differs from what we pass in Set/Update. Therefore I think it's not a bad idea to create examples.